k8s-api/pom.xml

```diff
@@ -14,6 +14,11 @@
       <groupId>io.enmasse</groupId>
       <artifactId>api-model</artifactId>
     </dependency>
+    <dependency>
+      <!-- required for string replacing -->
+      <groupId>org.keycloak</groupId>
+      <artifactId>keycloak-common</artifactId>
```

Comment:
I think it would be better to write our own/copy paste StringPropertyReplacer into the lib than adding a dependency on keycloak-common, which I suspect will pull in much more than we want.

Reply:
Actually what I had in mind doesn't work. So this dependency will be removed anyway.